File: labs/07 Access Control/web-login/php-apache/html/v3/break/break.php
<?php

$DICTIONARY = "words.txt";

// enter the DB hash below
$db_password = "";

// grab the dictionary words
$words = file($DICTIONARY, FILE_IGNORE_NEW_LINES);

// hash each dictionary word and compare
foreach ($words as $password)
{
    echo "Trying: $password\n";
    if (hash("sha256", $password) == $db_password)
    {
        // we found it!
        echo "**SUCCESS!\n";
        break;
    }
}

?>

File: labs/07 Access Control/web-login/php-apache/html/v3/break/break.py
from hashlib import sha256

DICTIONARY = "words.txt"

# enter the DB hash below
db_password = ""

# grab the dictionary words
with open(DICTIONARY, "r") as file:
    words = file.read().splitlines()

# hash each dictionary word and compare
for password in words:
    print(f"Trying: {password}")
    if (sha256(password.encode("utf-8")).hexdigest() == db_password):
        # we found it!
        print("**SUCCESS!")
        break

